crypto/objects/obj_dat.c

```diff
@@ -221,25 +221,45 @@ void ossl_obj_cleanup_int(void)
     objs_free_locks();
 }
 
-int OBJ_new_nid(int num)
+/*
+ * Requires that the ossl_obj_lock be held
+ * if TSAN_REQUIRES_LOCKING defined
+ */
+static int __OBJ_new_nid(int num)
```

Comment:
We never use double underscore prefix and IMO we should not start to. Please use `obj_new_nid_unlocked` as the function name instead.

Reply:
done